src/net.cpp

```diff
@@ -466,7 +466,7 @@ void CNode::PushVersion()
     else
         LogPrint("net", "send version message: version %d, blocks=%d, us=%s, peer=%d\n", PROTOCOL_VERSION, nBestHeight, addrMe.ToString(), id);
     PushMessage("version", PROTOCOL_VERSION, nLocalServices, nTime, addrYou, addrMe,
-                nLocalHostNonce, strSubVersion, nBestHeight, true);
+                nLocalHostNonce, strSubVersion, nBestHeight, !GetBoolArg("-blocksonly", DEFAULT_BLOCKSONLY));
```

Comment:
Checking my understanding, is this where we communicate to peers that they should not relay transactions to us?

Reply:
Yes:

https://github.com/zcash/zcash/blob/b08c673ddd0c61e95bc89149a732c16a263f59e9/src/main.cpp#L5656-L5659